fusedrug_examples/tests/test_design_amp.py

```diff
@@ -0,0 +1,29 @@
+import unittest
+import fusedrug_examples.design.amp.classifier.main_classifier_train as main_classifier_train
+import fusedrug_examples.design.amp.design.main_design_train as main_design_train
+from omegaconf import OmegaConf
+from pathlib import Path
+class DesignAMPTestCase(unittest.TestCase):
+
+    def test_classifier(self):
+        config_path = Path(__file__, "../../design/amp/classifier/config.yaml")
+        cfg = OmegaConf.load(config_path)
+        cfg.train.trainer_kwargs.max_epochs = 1
```

Comment:
Not sure how much the test takes
But if you case load just part of the data and not all of it.
You can do that be setting uniprot data path to None.

Reply:
done.